tabpy-server/server_tests/test_config.py

```diff
@@ -127,128 +152,97 @@ def setUp(self):
         self.addCleanup(os.chdir, self.cwd)
 
     def test_http(self):
-        self.fp.write(b'TABPY_TRANSFER_PROTOCOL = http')
+        self.fp.write("[TabPy]\n"
+                      "TABPY_TRANSFER_PROTOCOL = http".encode())
         self.fp.close()
```

Comment:
I guess unrelated to this PR but would be much better to call close in a teardown, or better yet use an io.StringIO instance which doesn't require explicit closing. If that doesn't work a still better option if you consider using other 3rd party testing tools like pytest could use a fixture to manage these tests.